src/test/streams_tests.cpp

```diff
@@ -9,8 +9,7 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
+
 using namespace boost::assign; // bring 'operator+=()' into scope
```

Comment:
Yeah, but you can't add onto it afterwards (using `+=`), at least AFAIK. 
I think a nice compromise is to move the `using namespace boost::assign` into the functions using it. I can do that if people prefer.